src/JWT.ts

```diff
@@ -423,7 +423,7 @@ export async function verifyJWT(
     }
     did = payload.did
   } else {
-    did = payload.iss
+    did = payload.iss || payload.client_id
```

Comment:
What if `iss` and `client_id` are both present and non equivalent.

Reply:
added an IF statement just for this kind of JWT

Reply:
> What if `iss` and `client_id` are both present and not equivalent.

According to the spec, `iss` shouldn't be present. So, if both `iss` and `client_id` are present, this proposed logic shouldn't be reached.
The tricky part here is that there is no a direct way to detect if payload belongs to a JWT Presentation Profile request.